tests/src/test/scala/invokerShoot/ShootInvokerTests.scala

```diff
@@ -172,6 +172,7 @@ class ShootInvokerTests extends TestHelpers with WskTestHelpers with JsHelpers w
 
   it should "add new parameters and annotations while copying an action" in withAssetCleaner(wskprops) {
     (wp, assetHelper) =>
+      val runtime = "nodejs:10"
```

Comment:
just make this `nodejs:default` and we'll change the default in the runtimes.json.
otherwise we'll be doing this again when node 10 goes out of style.

Reply:
Right, that would be the best 👍. 
Unfortunately many tests explicitly check the returned runtime string, e.g. for 'nodejs:6'. When we use 'nodejs:default' at action creation, this returned runtime string would be whatever the actual default runtime is.
In this case we would need to either query somehow in the test case what the current default runtime is, which we then can use to compare in the test case.
Or, we could check against a list of possible nodejs runtimes (nodejs:6, nodejs:8, nodejs:10). But then we would also need to adjust this list when we add newer versions.
Or, the other option would be, we do not compare the returned runtime string at all. Which would be the easiest solution,

What do you think?

Reply:
could those tests just check against the pattern `nodejs:*`?

Reply:
i'd think a more generic check is sufficient since there are unit tests that check that `:default` resolves to a specific version of a runtime.

Reply:
yes, makes sense, I will look for a generic check like `nodejs:*` and see if we can modify the tests accordingly … the travis 'System Tests' also still show a failure I need to analyse and fix… I will get back to you when I am done with that :-)